docs/chronologue/c_how_chronologue_works.md

```diff
@@ -16,35 +23,62 @@ Spacetime is when you combine the three spatial dimensions (3D) and time, result
+
+>Note: The following paragraphs contain fiction and do not reflect reality. <!--This part is redundant: and do not reflect reality--> 
 
 ## What is the Chronologue?
 
 Chronologue consists of two things:
 * The time-travel telescope
 * Software to change the parameters of the time-travel telescope
-
-A usual telescope allows you to see the light of stars that are many lightyears away. Because it takes a long time for the light to travel across the universe, we are not looking at the current state of this star, but the past. Let’s look at a more tangible example to clarify this phenomenon: 
-
+<!--
+Watch out for unparallel structures. 
+Change suggestion: 
+* A time-travel telescope. 
+* A software to change the parameters of the time-travel telescope.
+-->
+
+A usual telescope allows you to see the light of stars that are many lightyears away. Because it takes a long time for the light to travel across the universe, we are not looking at the current state of this star, but the past. Let’s look at a more tangible example to clarify this phenomenon: <!-- You can replace this sentence and the explanation with the following: Is like when you detect a lag in a live streaming service.--> 
+<!--Change suggestion for the first two sentences: 
+The light that reaches the earth has travelled across the universe for several, sometimes millions of years. This means that the light we observe with the help of telescopes is a snapshot of the past of its source star.
+--> 
+
+<!--I think this explanation is optional--> 
 You’re at home watching the football world cup via an internet stream and you notice the connection is sometimes a bit laggy. Suddenly you hear your neighbors cheer aloud but your screen only shows people chasing a football. Two seconds later, you also see that a team scored a goal. 
 
-The time-travel telescope is an upgraded, more complex telescope that lets you **look** at the past, the present, and the future in any location in the universe. You can access the time-travel telescope via your browser as a viewer and look at pre-programmed time travel destinations, such as the discovery of fire. If you need to access a specific time in space, you can do so by using the Chronologue API. 
+The time-travel telescope is an upgraded, more complex telescope that lets you **look** at the past <!--and also--> the present, and the future <!--. Moreover, the time-travel Chronologue telescope gives you access to any location in the universe-->. in any location in the universe. You can access the time-travel telescope via your browser as a viewer and look at pre-programmed time travel destinations, such as the discovery of fire. If you need to access a specific time in space, you can do so by using the Chronologue API. 
 
 <!--If you want to integrate Chronologue within your program, have a look at the Chronologue SDK. Active development is still up for discussion...-->
 
 ## How does the Chronologue telescope work?
```

Comment:
Wormholes make a good placeholder for now because almost everything else in that paragraph will be true about whatever we end up with. I think my ideas are beyond the scope of this review because they mainly concern other documents about the physical design and troubleshooting of the telescope. When I write those, I can edit this document to reflect any differences.